src/cascadia/TerminalApp/CommandPalette.xaml

```diff
@@ -205,6 +205,13 @@
                                   Visibility="{x:Bind Item.(local:TabPaletteItem.TabStatus).IsInputBroadcastActive, Mode=OneWay}" />
 
                     </StackPanel>
+
+                    <Ellipse Grid.Column="4"

```

Comment:
Slightly worried that this will overlap with the scrollbar on the tab picker if there's a _lot_ of tabs. Who knows - maybe that column was vestigial?

Reply:
It kind of overlaps, but it doesn't seem like a big problem to me 😅 

![resim](https://github.com/user-attachments/assets/21e7dbfa-e9d0-40cc-b144-3cacf322ed8c)